src/vs/platform/quickinput/browser/commandsQuickAccess.ts

```diff
@@ -310,6 +310,7 @@ export class CommandsHistory extends Disposable {
 
 	private static cache: LRUCache<string, number> | undefined;
 	private static counter = 1;
+	private static isDirty = false;
```

Comment:
I associate `dirty` more with editors, maybe `shouldSaveState`?

Reply:
Renamed to `hasChanges`